README.md

```diff
@@ -1,25 +1,103 @@
-Adding your package to conda-forge
-----------------------------------
+## Build status
+
+[![Circle CI](https://circleci.com/gh/conda-forge/staged-recipes/tree/master.svg?style=shield)](https://circleci.com/gh/conda-forge/staged-recipes/tree/master)
+
+[![Build Status](https://travis-ci.org/conda-forge/staged-recipes.svg?branch=master)](https://travis-ci.org/conda-forge/staged-recipes)
+
+[![Build status](https://ci.appveyor.com/api/projects/status/3lju80dibkmowsj5/branch/master?svg=true)](https://ci.appveyor.com/project/conda-forge/staged-recipes/branch/master)
```

Comment:
At some point in this convo, it was suggested these be put on one line. Definitely in favor of that.

Though could also see an argument for only keeping the badge associated with feedstock conversion (currently Travis CI) as the others don't have any real meaning. This seems fine to me too.